Makefile

```diff
@@ -12,11 +12,16 @@ install-pre-commit:
 test:
 	python -m unittest
 
+test-unit:
```

Comment:
can you change this one to unit

Reply:
Do we want linter checks too, or just keep it simple?